src/test/java/roomescape/MemberLogicTest.java

```diff
@@ -0,0 +1,104 @@
+package roomescape;
+
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.annotation.DirtiesContext.ClassMode;
+import roomescape.exception.CreateMemberFailException;
+import roomescape.member.controller.MemberController;
+import roomescape.member.dto.MemberRequest;
+import roomescape.member.dto.MemberResponse;
+import roomescape.member.service.MemberService;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@DirtiesContext(classMode = ClassMode.AFTER_EACH_TEST_METHOD)
+public class MemberLogicTest {
+
+    @LocalServerPort
+    private int port;
+
+    @BeforeEach
+    void setUp() {
+        RestAssured.port = port;
+    }
+
+    @Autowired
+    private MemberController memberController;
+
+    @Autowired
+    private MemberService memberService;
+
+    @Test
+    @DisplayName("아이디_생성_테스트")
+    void createMemberTest() {
+        // given
+        MemberRequest memberRequest = new MemberRequest("Doyo", "member@example.com", "password");
+
+        // when
+        ResponseEntity<MemberResponse> response = memberController.createMember(memberRequest);
+
+        // then
+        assertThat(response.getStatusCode()).isEqualTo(HttpStatus.CREATED);
+        assertThat(response.getBody()).isNotNull();
+        MemberResponse responseBody = response.getBody();
+        assertThat(responseBody.getId()).isNotNull();
+        assertThat(responseBody.getName()).isEqualTo("Doyo");
+        assertThat(responseBody.getEmail()).isEqualTo("member@example.com");
+    }
+
+    @Test
+    @DisplayName("잘못된_이메일_형식_테스트")
+    void createMemberWithInvalidEmailTest() {
+        // given & when & then
+        assertThrows(CreateMemberFailException.class, () -> {
+            new MemberRequest("Doyo", "invalid-email", "password");
+        });
```

Comment:
```suggestion
        assertThrows(CreateMemberFailException.class, () -> new MemberRequest("Doyo", "invalid-email", "password"));
```

알고 계시겠지만, 한줄짜리 람다식의 바디는 이렇게 표현이 가능해요~

Reply:
람다식에 대해서 좀 더 학습해야 할 것 같네요. 감사합니다 😃